docs/tutorials/security/part-3.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Part 3: User trust & management'
+authors: 'Benjamin Fuentes (Marigold)'
+last_update:
+  date: 11 January 2024
+---
+
+> Note : clone this [project](https://github.com/marigold-dev/training-security-3.git) for compiling and testing this tutorial
```

Comment:
This repo appears to be empty except for a license and readme.

Reply:
yes, finally I did not pushed code on part 3. A bit too long, but feasible if we have more time and request from users